README.md

```diff
@@ -34,6 +34,12 @@ Default: `tcp`
 
 Set to `udp` to proxy in UDP mode.
 
+### `MAX_CONNECTIONS`
+
+Default: `100`
+
+Limits the maximum number of accepted connections at one time per port.
+
```

Comment:
```suggestion
You can set it to `4194304` to set it to unlimited (See https://stackoverflow.com/questions/6294133/maximum-pid-in-linux)
```

Reply:
I'll add a note matching this. however just to be clear this would probably need to be set on the docker host system (i'll investigate) and it is far from unlimited in the mathematical sense of numbers.

Reply:
Ok :+1: 
Yes, that was only a suggestion, it can be phrased better :smile: